src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -51,7 +51,7 @@ export type AutocompleteRendererOptions = {
   /**
    * Searches into the indices with the provided query.
    */
-  refine: (query: string) => void;
+  refine: Refine<string>;
```

Comment:
the name "query" is now gone from autocomplete, is that correct?

```
refine(
    ^^^query: string
```

Reply:
Correct.

Reply:
Tried to find a solution but couldn't :( https://www.typescriptlang.org/play/index.html#code/C4TwDgpgBAZgdgHgCoDkCGBbCA+KBeKACgGcB7LAcQjggCcBLAY3SwC4pjgG4BzASny4AbqXoATAFASA9NKjAAFvWJRlcAOTAoQtABtxMuaEixEqTDnxEA2uawBddp279B20ZInHo9OJzRwjNAE8AjqGCAAShBgumhBWHDALBDq2FKMpH5aAB7svv6BwUQR0bHxEInJFgJ4uACucGIQML4QnjmE6gCSUADuygryCtBo9cCkmRixEMDQmfW6YlA89ELQpTFxCdTVWKrZEGjLpDAc5BBUNAzMFup8UrJQnQ9AA

Reply:
I think it's fine, 👍